lib/bloc/trending_movies_bloc.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
+
+import '../resources/trending_movies_repository.dart';
+import '../interfaces/ITrendingMoviesBloc.dart';
+import '../models/item_model.dart';
+
+class TrendingMoviesBloc extends ITrendingMovies{
+  final trendingMoviesRepository = TrendingMoviesRepository();
+  StreamController<ItemModel> movieStream = StreamController();
+  @override
+  void dispose() {
+    movieStream.close();
+  }
+
+  @override
+  void fetchTrendingMovies() async {
+    final trendingMovies = await trendingMoviesRepository.fetchTrendingMovies();
+    movieStream.sink.add(trendingMovies);
+  }
+
+  @override
+  Future<void> initialize() {}
+
+  @override
+  Stream<ItemModel> get stream => movieStream.stream;
+
+}
```

Comment:
Remember to use ctrl+alt+l, you missed a newline at end of file here.

Reply:
Done